runtime/compiler/env/j9method.cpp

```diff
@@ -6175,7 +6199,6 @@ TR_ResolvedJ9Method::stringConstant(I_32 cpIndex)
 void *
 TR_ResolvedJ9Method::methodTypeConstant(I_32 cpIndex)
    {
-   TR_ASSERT(!TR::CompilationInfo::getStream(), "no server");
```

Comment:
These asserts were put because we didn't handle all possible frontend queries in JITServer. We started with the ones that were hit and as we ran more and more code, we hit and more and implemented those as well. If we remove these asserts we need to implement their queries.
Until then the asserts need to stay surrounded by `#ifdef JITSERVER_SUPPORT`